portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -517,6 +512,48 @@ definition {
 		}
 	}
 
+	@description = "LPS-159013 ES module custom element portlets render incorrect <script> node when dragged in the UI"
+	@priority = "4"
+	test CustomElementRendersCorrectlyWhenPlacedOnWidgetPage {
+		property portal.acceptance = "true";
+		property testray.component.names = "Frontend Dataset,Remote Apps";
```

Comment:
Only Remote Apps is fine, nothing unique to add in test coverage for FDS from this test from what I can tell.